pom.xml

```diff
@@ -229,6 +259,14 @@
                   <version>[1.7,)</version>
                 </requireJavaVersion>
                 <requireUpperBoundDeps/>
+                <banDuplicateClasses>
```

Comment:
Actually, can we separate this into a separate PR so it'll appear as its own entry in the changelog.

Reply:
SG